### PR TITLE
Adding renewal emails and templates

### LINKED ISSUE #201: Implement Lewis Library Locker Renewals
### User story
As a Lewis Library locker user who is returning next semester I would like a renewal email reminding to renew my locker agreement. 

### Implementation notes, if any

- [ ] Sends a renewal email to all eligible current locker users (everyone but a graduating senior).
- [ ] Create lewis specific renewal message template.

### PR DESCRIPTION
**Pull request by rladdusaw**
Resolves #201

### CODE REVIEW

File: app/views/user_mailer/lewis_locker_renewal.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<p>Hi,</p>
```
Comment (kevinreiss) [inline app/views/user_mailer/lewis_locker_renewal.html.erb:1]: Sylvia asks for this text to be used here:
```
Your locker privileges expire on <%= @assignment.expiration_date %>. If you would like to renew your locker, please email lewislib@Princeton.edu Please remember if your status changes during the year from enrolled to on leave, in absentia or terminated, you will need to vacate your locker.
 
If you will not be renewing your locker, please let me know. I can make arrangements for your library materials to be returned.
   
Please contact me if you have any questions or problems. Thank you.
   
Sylvia Swain
Lewis Library Coordinator258-3267

Your locker privileges expire on <%= @assignment.expiration_date %>. If you would like to renew your locker, please email [lewislib@Princeton.edu](mailto:lewislib@Princeton.edu) Please remember if your status changes during the year from enrolled to on leave, in absentia or terminated, you will need to vacate your locker.
 
If you will not be renewing your locker, please let me know. I can make arrangements for your library materials to be returned.
Please contact me if you have any questions or problems. Thank you.
Sylvia Swain
Lewis Library Coordinator
609-258-3267

File: app/views/user_mailer/lewis_locker_renewal.text.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Hi,
```
Comment (kevinreiss) [inline app/views/user_mailer/lewis_locker_renewal.text.erb:1]: Use the text as my earlier comment.

**Review by kevinreiss — CHANGES_REQUESTED** [review]

File: config/environments/development.rb [hunk 3]
```diff
@@ -37,12 +37,18 @@
...
   config.action_mailer.perform_caching = false
 
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
 
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'localhost',
+    port: 1025
+  }
+
```
Comment (sandbergja) [inline config/environments/development.rb:51]: Thanks for adding this!

**Review by sandbergja — APPROVED** [review]
🎉  Looks great, thanks @rladdusaw !

**Pull request merged**